File: beginner/1010.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var line1 = lines.shift().split(' ');
var cod1 = parseInt(line1[0]);
var num1 = parseInt(line1[1]);
var val1 = parseFloat(line1[2]);

var line2 = lines.shift().split(' ');
var cod2 = parseInt(line2[0]);
var num2 = parseInt(line2[1]);
var val2 = parseFloat(line2[2]);

var total = (num1 * val1) + (num2 * val2);

console.log('VALOR A PAGAR: R$ ' + total.toFixed(2));

File: beginner/1010.c
#include <stdio.h>

int main() {
 
	int cod1, cod2, peca1, peca2;
	float preco1, preco2;
	
	scanf("%d %d %f", &cod1, &peca1, &preco1);
	scanf("%d %d %f", &cod2, &peca2, &preco2);
	
	printf("VALOR A PAGAR: R$ %.2f\n",(peca1 * preco1) + (peca2 * preco2));
	return 0;
}
